linear_algebra/code/CMakeLists.txt

```diff
@@ -48,30 +48,17 @@ include_directories(include test .)
 
 add_executable(la_test ${Sources} ${Includes})
 
-set(CMAKE_VERBOSE_MAKEFILE 1)
-
-if(CXX_COMPILER STREQUAL clang++)
-    set(CMAKE_C_COMPILER   clang)
-    set(CMAKE_CXX_COMPILER clang++)
-else()
-    set(CMAKE_C_COMPILER   gcc)
-    set(CMAKE_CXX_COMPILER g++)
-endif()
-
-if(CMAKE_CXX_COMPILER STREQUAL clang++)
-
-    set(CMAKE_C_FLAGS_DEBUG   "-g -Wall -pedantic -Wextra")
```

Comment:
Just a note to say explicitly setting "-g" or "-03" for debug or release is not necessary as this is the default for these compiler modes. Please see build types here: http://www.brianlheim.com/2018/04/09/cmake-cheat-sheet.html